addons/binding/org.openhab.binding.pioneeravr/src/main/java/org/openhab/binding/pioneeravr/protocol/utils/VolumeConverter.java

```diff
@@ -41,9 +44,9 @@ public static double convertFromIpControlVolumeToDb(String ipControlVolume) {
      * @param volumeDb
      * @return the volume for IpControlRequest
      */
-    public static String convertFromDbToIpControlVolume(double volumeDb) {
-        double ipControlVolume = ((MIN_DB_VOLUME + volumeDb) * 2d) + 1d;
-        return formatIpControlVolume(ipControlVolume);
+    public static String convertFromDbToIpControlVolume(double volumeDb, int zone) {
+        double ipControlVolume = ((MIN_DB_VOLUME[zone - 1] + volumeDb) * 2d) + 1d;
```

Comment:
thanks for reviewing...

i added `validateZone` method in `volumeConverter` to prevent `ArrayIndexOutOfBoundsException`